libcloud/compute/drivers/openstack.py

```diff
@@ -2874,7 +2874,7 @@ def _to_port(self, element):
                 allowed_address_pairs=element['allowed_address_pairs'],
                 binding_vnic_type=element['binding:vnic_type'],
                 device_id=element['device_id'],
-                description=element['description'],
+                description=element.get('description', None),
```

Comment:
If you get a chance it would also be good to add a test case for it + changelog entry.